### PR TITLE
DCOS-43348: create a design token distribution

### PR DESCRIPTION
**Pull request by mperrotti**
I'm using [style-dictionary](https://github.com/amzn/style-dictionary/) to create a distribution of design tokens (design attributes that can be expressed in a key-value pair).

As a proof-of-concept, I've updated `color.stories.ts` to use design tokens.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,15 +15,17 @@
...
     "commitlint": "commitlint -e",
     "create:component": "./scripts/create-component",
     "deploy:storybook": "storybook-to-ghpages",
     "predist": "npm run clean",
-    "dist": "tsc -p tsconfig.dist.json",
+    "dist": "tsc -p tsconfig.dist.json && npm run build:design-tokens",
     "lint": "tslint -c tslint.json -p tsconfig.json",
     "setup:git-hooks": "./scripts/pre-install",
-    "start": "NODE_ENV=development npm run storybook --root ./ ",
+    "start": "NODE_ENV=development npm run build:design-tokens && npm run storybook --root ./ ",
```
Comment (mperrotti) [inline package.json]: Yes, this is weird, but we can just import design tokens as a package once we use Lerna to manage multiple packages in this repo.
Then we can remove all the design token changes in here

**Comment by mperrotti** [comment]
@GeorgiSTodorov - the tokens should be usable in dcos-ui. The style utilities will require us to set up Emotion in dcos-ui

**Comment by GeorgiSTodorov** [comment]
@mperrotti Please look into file conflict so that we can review.

File: packages/design-tokens/scripts/formats.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const SD_lessVariables = require("../../../node_modules/style-dictionary/lib/common/formats")[
```
Comment (nLight) [inline packages/design-tokens/scripts/formats.js]:
```suggestion
const SD_lessVariables = require("style-dictionary/lib/common/formats")[
```
also worked for me

File: packages/design-tokens/scripts/formats.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+const dateHeader = () => `
+  /**
+   * Do not edit directly
+   * Generated on ${new Date()}
+   */
+`;
+
+const lessVariables = {
+  name: "less/variablesCustom",
```
Comment (nLight) [inline packages/design-tokens/scripts/formats.js]: Is there a specific name we could give it? What is custom about our way of formatting it?

File: packages/design-tokens/scripts/formats.js (hunk 2 above)
Comment (nLight) [reply packages/design-tokens/scripts/formats.js]: I would still call it `lessVariables`

File: packages/design-tokens/scripts/formats.js (hunk 3 above)
Comment (mperrotti) [reply packages/design-tokens/scripts/formats.js]: I meant to remove this. I was doing a trying a slightly different format than style-dictionary gives you by default

**Pull request merged**